src/components/ExpandableCard.js

```diff
@@ -48,12 +52,24 @@ const Text = styled(motion.div)`
 
 const Question = styled.div`
   margin-right: 1rem;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    margin-right: 0;
+    margin-bottom: 0.5rem;
+  }
 `
 
 const ButtonContainer = styled.div`
   margin-left: 1rem;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    margin-left: 0;
+  }
 `
 
+const Link = styled(FakeLink)`
```

Comment:
We should be explicit here that this isn't an actual link.

```suggestion
const StyledFakeLink = styled(FakeLink)`
```